server/repository/src/test_db/sqlite.rs

```diff
@@ -38,83 +70,78 @@ pub(crate) async fn setup_with_version(
     inserts: MockDataInserts,
 ) -> (StorageConnectionManager, MockDataCollection) {
     let db_path = db_settings.connection_string();
+    let memory_mode = db_path.starts_with("file:");
+    if memory_mode || env_msupply_no_test_db_template() {
+        return setup_with_version_no_template(db_settings, version, inserts).await;
+    }
```

Comment:
This is the main change, changes below are mostly from changing the level of indentation...